packages/smart-wallet/test/test-invitation1.js

```diff
@@ -151,6 +155,7 @@ test.serial('handle failure to create invitation', async t => {
   const invitationIssuer = powers.issuer.consume.Invitation;
   const address = 'agoric1234';
 
+  // @ts-expect-error It's a promise.
   const walletsStorage = E(chainStorage).makeChildNode('wallet');
```

Comment:
```suggestion
  // @ts-expect-error Test setup ensures that chainStorage resolution is not undefined.
  const walletsStorage = E(chainStorage).makeChildNode('wallet');
```

another approach is something like:

```js
 * @param {BootstrapPowers & NonNullChainStorage} powers
 * @typedef {PromiseSpaceOf<{ chainStorage: StorageNode }>} NonNullChainStorage
```

Reply:
I think either of the above is good for this PR. You could also reference,
- https://github.com/Agoric/agoric-sdk/issues/8247

Reply:
added